gulpfile.js

```diff
@@ -75,6 +75,7 @@ const default_plugins = [
   'editing',
   'openrouteservice',
   'qplotly',
+  'qprocessing', //@since 3.9.0
```

Comment:
@volterra79 this is not a core plugin 👉 [g3w-admin-processing](https://github.com/g3w-suite/g3w-admin-processing)

Reply:
@Raruto you are right. Please remove it